### PR TITLE
Loadouts

### PR DESCRIPTION
**Pull request by Axionyxx**
## About the PR
Existing loadouts reviewed:
- [ ] TODO 

Additonal changes
- Split mail carrier and courier
- Removed useless remaining senior role prototypes

## Media

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

**Comment by Axionyxx** [comment]
This is now working in game, minus localization strings

File: Resources/Locale/en-US/job/job-description.ftl [hunk 1]
```diff
@@ -45,8 +45,4 @@ job-description-security = Catch criminals and enemies of the station, enforce t
 job-description-serviceworker = Learn the basics of bartending, cooking, and growing plants.
 job-description-visitor = Enjoy your visit to the station.
 job-description-warden = Patrol the security department, ensure that no one is stealing from the armory, and make sure that all prisoners are processed and let out when their time is up.
-job-description-zookeeper = Put on a joyful display of cute animals and space carps for all the crew to see. Currently unavailable.
-job-description-senior-engineer = Teach new engineers the basics of the station's engine, repairing, atmospherics and power.
-job-description-senior-researcher = Teach new scientists the basics of item printing, artifact research and anomalous objects.
-job-description-senior-physician = Teach new medics the basics of tending to the wounded, chemistry, diagnosing the diseased and disposing of the dead.
-job-description-senior-officer = Teach new officers the basics of searches, performing arrests, prison times and how to properly shoot a firearm.
+job-description-zookeeper = Put on a joyful display of cute animals and space carps for all the crew to see. Currently available on Gemini Station.
```
Comment (IamVelcroboy) [inline Resources/Locale/en-US/job/job-description.ftl:48]: Should maybe change to `Currently available on Submarine Station.`

Comment (rosieposieeee) [reply Resources/Locale/en-US/job/job-description.ftl:48]: probably a bad idea to mention any station, considering other maps might add the role

Comment (Axionyxx) [reply Resources/Locale/en-US/job/job-description.ftl:48]: Out of the scope for this pr

Comment (IamVelcroboy) [reply Resources/Locale/en-US/job/job-description.ftl:48]: Either way. Can always update if another station adds it but leaving it out is less future edits. 🤷‍♂️

**Comment by Axionyxx** [comment]
It is currently passing the linter, haven't ran other tests yet. Now all that's left is to review inherited loadouts

**Comment by Axionyxx** [comment]
Copied from Discord: I consider the current loadouts for our custom jobs to be mostly "done", except if anyone else has any feedback to offer in [#⁠loadouts](https://discord.com/channels/968983104247185448/1232025191194951792). I'll likely move to the step of checking all upstream loadouts tomorrow or the day after that

**Comment by pissdemon** [comment]
#1144

**Comment by VMSolidus** [comment]
Well, I can't say that I'm surprised by this. But this is where I guess I'll be permanently parting ways with making or maintaining any code for DeltaV. My own fork doesn't have this version of loadouts, and never will. And this touches so much code that once its merged, we fundamentally won't be compatible forks anymore. We're merging a different version of loadouts, that has MRP and HRP servers first in its design choices. I'm really sad to say it, but this is it for me.

**Comment by Axionyxx** [comment]
Please quit merging master on prs that don't need it. Just because the button is there doesn't mean you need to press it. If it isn't your pr, don't touch it.

The only exception being when you want to merge something and need the latest tests ran.

**Comment by Colin-Tel** [comment]
> Please quit merging master on prs that don't need it. Just because the button is there doesn't mean you need to press it. If it isn't your pr, don't touch it.

Negative, I was attempting resolving the merge conflict. I do understand you, however. Going forward I won't be doing this for other PRs.

**Comment by Axionyxx** [comment]
> > Please quit merging master on prs that don't need it. Just because the button is there doesn't mean you need to press it. If it isn't your pr, don't touch it.
> 
> Negative, I was attempting resolving the merge conflict. I do understand you, however. Going forward I won't be doing this for other PRs.

Why do you need to resolve the conflict on a pr that isn't yours? Don't worry too much about it, its fine. Just don't in the future for stuff like this

**Comment by Colin-Tel** [comment]
> Why do you need to resolve the conflict on a pr that isn't yours?

Because I felt it's my fault. Like I said, I won't be doing this going forward.

**Comment by IamVelcroboy** [comment]
> > > Please quit merging master on prs that don't need it. Just because the button is there doesn't mean you need to press it. If it isn't your pr, don't touch it.
> > 
> > 
> > Negative, I was attempting resolving the merge conflict. I do understand you, however. Going forward I won't be doing this for other PRs.
> 
> Why do you need to resolve the conflict on a pr that isn't yours? Don't worry too much about it, its fine. Just don't in the future for stuff like this

Probably because this touches all the map files. Which, honestly, should probably be done in a separate PR because otherwise we are going to have a ton of conflicts/lost work as mappers update their maps

**Comment by IamVelcroboy** [comment]
My bad...I see now, it's the proto-files not the actual map files (GH on phone moment)

**Comment by Axionyxx** [comment]
I'll undraft when tests stop failing 100% of the time. Should be in a mostly ready state now

**Comment by Axionyxx** [comment]
Tests pass locally, I just need to wait for actions to work again.

**Review by deltanedas — APPROVED** [review]
mimic test fail irrelevant

**Pull request merged**